launcher/src/backend/TekuGasLimitConfig.js

```diff
@@ -0,0 +1,23 @@
+export class TekuGasLimitConfig{
+  constructor(nodeConnection) {
+    this.nodeConnection = nodeConnection;
+  }
+
+  async createGasConfigFile(gasLimit, feeRecipient, configPath){
+    await this.nodeConnection.sshService.exec(`cat > ${configPath}/gas_config.json`);
+    await this.nodeConnection.sshService.exec(`echo '{\n\t"default_config":\n\t{\n\t\t"fee_recipient":"${feeRecipient}",\n\t\t"builder":\n\t\t{\n\t\t\t"enabled":true,\n\t\t\t"gas_limit":"${gasLimit}"\n\t\t}\n\t}\n}' > ${configPath}/gas_config.json`);
```

Comment:
how about doing more readable like creating json format ---> "const configContent = `{
      "default_config": {
        "fee_recipient": "${feeRecipient}",
        "builder": {
          "enabled": true,
          "gas_limit": "${gasLimit}"
        }
      }
    }`;

    await this.nodeConnection.sshService.exec(`echo '${configContent}' > ${configPath}/gas_config.json`);"
    for better readability also for maintaining

Reply:
we should have quite some functions you can use to escape everything for the shell. see [escapeStringForShell‎](https://github.com/stereum-dev/ethereum-node/blob/main/launcher/src/backend/StringUtils.js#L6)